juriscraper/OpinionSiteLinear.py

```diff
@@ -158,6 +159,10 @@ def _get_attorneys(self):
         """Goes into OpinionCluster.attorneys, type: string"""
         return self._get_optional_field_by_id("attorney")
 
+    def _get_headnotes(self):
```

Comment:
drop headnotes from this PR ... 

and we should add them in a different pr if we want to include headnotes